include/picongpu/simulation/stage/SynchrotronRadiation.hpp

```diff
@@ -113,188 +118,180 @@ namespace picongpu::simulation::stage
-            constexpr float_64 minZqExp = InterpolationParams::minZqExponent;
-            constexpr float_64 maxZqExp = InterpolationParams::maxZqExponent;
-            constexpr float_64 tableEntries = InterpolationParams::numberTableEntries;
-
-            //! precompute F1 and F2 on log scale
-            for(uint32_t iZq = 0; iZq < tableEntries; iZq++)
-            {
-                /** inverse function for index retrieval:
-                 * index = (log2(zq) - minZqExp) / (maxZqExp - minZqExp) * (tableEntries-1);
-                 */
-                float_64 zq = std::pow(2., minZqExp + (maxZqExp - minZqExp) * iZq / (tableEntries - 1));
+            /** inverse function for index retrieval:
+             * index = (log2(zq) - minZqExp) / (maxZqExp - minZqExp) * (tableEntries-1);
+             */
+            float_64 zq = std::pow(2., minZqExp + (maxZqExp - minZqExp) * iZq / (tableEntries - 1));
 
-                float_64 const F1 = firstSynchrotronFunction(zq);
-                float_64 const F2 = secondSynchrotronFunction(zq);
+            float_64 const F1 = firstSynchrotronFunction(zq);
+            float_64 const F2 = secondSynchrotronFunction(zq);
 
-                setTableValuesF1F2(iZq, Accessor::f1, static_cast<float_X>(F1));
-                setTableValuesF1F2(iZq, Accessor::f2, static_cast<float_X>(F2));
-            }
-            //! move the data to the device
-            tableValuesF1F2->hostToDevice();
+            setTableValuesF1F2(iZq, Accessor::f1, static_cast<float_X>(F1));
+            setTableValuesF1F2(iZq, Accessor::f2, static_cast<float_X>(F2));
         }
+        //! move the data to the device
+        tableValuesF1F2->hostToDevice();
+    }
 
-        //! Mapping for kernels
-        MappingDesc cellDescription;
-
-        /** precomputed first and second synchrotron functions:
-         *  -> 2d grid of floats_64 -> tableValuesF1F2[zq][0/1] ; 0/1 = F1/F2
-         */
-        std::shared_ptr<GridBuffer<float_X, 2>> tableValuesF1F2;
-        //! used for table access
-        enum struct Accessor : uint32_t
-        {
-            f1 = 0u,
-            f2 = 1u
-        };
-        constexpr uint32_t u32(Accessor const t)
-        {
-            return static_cast<uint32_t>(t);
-        }
+    //! Mapping for kernels
+    MappingDesc cellDescription;
 
-        /** flag to check if the requirements 1 and 2 are met
-         * We check the requirements in class SynchrotronIdea() in:
-         * picongpu/include/picongpu/particles/synchrotron/AlgorithmSynchrotron.hpp
-         */
-        std::shared_ptr<GridBuffer<int32_t, 1>> failedRequirementQ;
-        bool failedRequirementPrinted;
+    /** precomputed first and second synchrotron functions:
+     *  -> 2d grid of floats_64 -> tableValuesF1F2[zq][0/1] ; 0/1 = F1/F2
+     */
+    std::shared_ptr<GridBuffer<float_X, 2>> tableValuesF1F2;
+    //! used for table access
+    enum struct Accessor : uint32_t
+    {
+        f1 = 0u,
+        f2 = 1u
     };
```

Comment:
according to the error message of the CI, you are a missing a `;`after the class

Reply:
done